Gemfile

```diff
@@ -62,6 +62,9 @@ gem "sshkey",                         "~>1.8.0",   :require => false
 gem "uglifier",                       "~>2.7.1",   :require => false
 gem "websocket-driver",               "~>0.6.3"
 
+# Required by the oVirt 4 provider:
+gem "ovirt-engine-sdk", "~>4.0.0.alpha", :require => false
```

Comment:
I think this should be moved up (alphabetcial order) and the comment at the end of the line